build.gradle

```diff
@@ -126,38 +148,69 @@ minecraft {
             }
         }
 
-        testClient {
+        gameTestClient {
             workingDirectory project.file('test-files/client')
             parent runs.client
 
+            def oldClasspath = lazyTokens["minecraft_classpath"]
+            lazyToken("minecraft_classpath") {
+                // Add all files in testMinecraftLibrary to the classpath.
+                def allFiles = new HashSet<String>();
+
+                def minecraftClasspath = oldClasspath?.get()
+                if (!minecraftClasspath != null && !minecraftClasspath.isEmpty()) allFiles.addAll(minecraftClasspath.split(File.pathSeparatorChar))
```

Comment:
Well, I mean, is **`!`** `minecraftClasspath != null` a correct syntax? I checked gradle docs and the syntax should same as java?

Reply:
Ahhh that's why intellij showed me a warning 
Didn't saw that, oops